packages/rocketchat-theme/server/colors.less

```diff
@@ -911,3 +911,50 @@ label.required::after {
 		opacity: inherit;
 	}
 }
+
+/** ----------------------------------------------------------------------------
+ * Input Range Slider
+ */
+
+.range-slider__range {
```

Comment:
We already have classes for these colors, can you add those classes on your template?